calico/felix/fetcd.py

```diff
@@ -395,6 +397,7 @@ def _dispatch_msg_from_driver(self, msg_type, msg):
             # Yield to ensure that other actors make progress.
             # Sleep must be non-zero to work around gevent
             # issue where we could be immediately rescheduled.
+            #PLW: Why do we need this?
```

Comment:
If the events are coming in quickly enough, the socket never blocks and this code loops without yielding.  I expected the recv() call to yield anyway but it doesn't seem to.  I only added this after I saw the other actors get starved when we're under high churn.